### DISCUSSION TITLE
MIDI CC for OSC change?

### DISCUSSION THREAD

**Discussion by BKLronin** [bangcorrupt/hacktribe#243] [Q&A] [answered]
Is it possible to change the OSC type via MIDI CC message? I can´t really get trough to the E2 with any cc message from Bitwig.

**Comment by bangcorrupt** [comment, marked as answer]
Not yet but I will add it at some point, hopefully motion sequencing too.  For now you would need to read, modify, write the pattern file via sysex.  I think this interrupts the audio.

**Reply by BKLronin** [reply]
That would be neat as we could export a wide variety of sounds from the synth that way. 

I have a hard time to leave the synth firmware. Listened to some nice patterns today and just taking the samples with me would be enough at least for me (Drums).
